Makefile

```diff
@@ -0,0 +1,9 @@
+init:
+    pip install pipenv
+    pipenv install
+
+test:
+    echo "Tests Pending"
```

Comment:
Alright, I went with the minimum option. We can change it later if we need to.

One thing I am concerned about, is that Windows users may be coming across the following error when running tests on their local machine: 

https://www.google.com/search?q=OSError%3A+raw+write()+returned+invalid+length+2+(should+have+been+between+0+and+1)&oq=OSError%3A+raw+write()+returned+invalid+length+2+(should+have+been+between+0+and+1)&aqs=chrome..69i64j0j5.1444j0j9&sourceid=chrome&ie=UTF-8

Hopefully it will be fixed soon.

Reply:
I'm confused.  Are the tests failing on your machine?  If so, please share the full output of the tests.

Reply:
Tests pass if they get to run. If they do not run, it is because of the OSError below.

```
...........................Traceback (most recent call last):
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\__main__.py", line 18, in <module>
    main(module=None)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\main.py", line 94, in __init__
    self.runTests()
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\main.py", line 255, in runTests
    self.result = testRunner.run(self.test)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\runner.py", line 176, in run
    test(result)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\suite.py", line 122, in run
    test(result)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\suite.py", line 122, in run
    test(result)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\suite.py", line 122, in run
    test(result)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 648, in __call__
    return self.run(*args, **kwds)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\case.py", line 616, in run
    result.addSuccess(self)
  File "C:\Users\Don\AppData\Local\Programs\Python\Python35-32\lib\unittest\runner.py", line 64, in addSuccess
    self.stream.flush()
OSError: raw write() returned invalid length 2 (should have been between 0 and 1)
```